src/content/docs/cheerpj2/05-reference/Runtime-API.md

```diff
@@ -216,7 +202,21 @@ cheerpjInit({
 });
 ```
 
-## cheerpjCreateDisplay
+#### `enablePreciseAppletArchives`
+
+Boolean option which enables a special applet classloader.
```

Comment:
... which makes the classloader behavior more similar to the native Java plugin. (consider mentioning that classloader behavior is not accurate in CJ2 and people should consider upgrading to CJ3)